metabase/metabase
# Refactor `TableSimple` visualization

Cleans up `TableSimple` visualization component:

* breaks down into a few smaller components
* converts to a functional component
* migrates to styled-components (not completely, but most of the styles are in emotion now)
* no TypeScript yet

### To Verify

1. Create a question visualized as a table
2. Add it to dashboard
3. Ensure the table looks as usually
4. Ensure column sorting works as before
5. Ensure pagination works as before
